src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,40 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  movie: Movies;
+}
+export const MovieCard: React.FC<Props> = ({ movie }) => (
```

Comment:
```suggestion
}

export const MovieCard: React.FC<Props> = ({ movie }) => (
```
Давай будемо якось візуально розділяти логічні блоки нашого коду